examples/06-verif-manual/vm-007-plastic_compression_of_a_pipe_assembly.py

```diff
@@ -0,0 +1,578 @@
+   as shown below. Determine the load-deflection curve of the assembly
+   as it is compressed into the plastic region by an axial displacement.
+   Assume that the end plates are so stiff that both tubes are shortened by
+   exactly the same amount.
+
+Reference:
+ - S. H. Crandall, N. C. Dahl, An Introduction to the Mechanics of Solids,
+   McGraw-Hill Book Co., Inc., New York, NY, 1959, pg. 180, ex. 5.1.
+
+Analysis Type(s):
+ - Static, Plastic Analysis (``ANTYPE=0``)
+
+Element Type(s):
+ - Plastic Straight Pipe Element (PIPE288)
+ - 4-Node Finite Strain Shell (SHELL181)
+ - 3-D Structural Solid Elements (SOLID185)
+
+.. image:: ../../_static/vm7_setup.png
+   :width: 400
+   :alt: VM7 Problem Sketch
+
+.. image:: ../../_static/vm7_setup_2.png
+   :width: 400
+   :alt: VM7 Finite Element Models
+
+Material Properties
+ - :math:`E_{\mathrm{s}} = 26875000 psi`
+ - :math:`\sigma_{\mathrm{(yp)s}} = 86000 psi`
+ - :math:`E_{\mathrm{a}} = 11000000  psi`
+ - :math:`\sigma_{\mathrm{(yp)a}} = 55000 psi`
+ - :math:`\nu = 0.3`
+
+Geometric Properties:
+ - :math:`l = 10  in`
+ - :math:`A_{\mathrm{s}} = 7 in^2`
+ - :math:`A_{\mathrm{a}} = 12 in^2`
+
+Loading:
+ - 1st Load Step: :math:`\delta = 0.032 in`
+ - 2nd Load Step: :math:`\delta = 0.050 in`
+ - 3rd Load Step: :math:`\delta = 0.100 in`
+
+Analysis Assumptions and Modeling Notes:
+ - The following tube dimensions, which provide the desired cross-sectional
+   areas, are arbitrarily chosen.
+   Inner (steel) tube: inside radius = 1.9781692 in.,
+   wall thickness = 0.5 in.
+   Outer (aluminum) tube: inside radius = 3.5697185 in.,
+   wall thickness = 0.5 in.
+
+ - The problem can be solved in three ways:
+   • using PIPE288 - the plastic straight pipe element
+   • using SOLID185 - the 3-D structural solid element
+   • using SHELL181 - the 4-Node Finite Strain Shell
+
+ - In the SOLID185 and SHELL181 cases, since the problem is axisymmetric,
+   only a one element :math:`\Theta` -sector is modeled. A small angle :math:`\Theta = 6°`
+   is arbitrarily chosen to reasonably approximate the circular boundary
+   with straight sided elements.
+   The nodes at the boundaries have the UX (radial) degree of freedom coupled.
+   In the SHELL181 model, the nodes at the boundaries additionally have
+   the ROTY degree of freedom coupled.
+
+"""
+# sphinx_gallery_thumbnail_path = '_static/vm7_setup.png'
+
+###############################################################################
+# Start MAPDL
+# ~~~~~~~~~~~
+# Start MAPDL and import additionally Numpy and Pandas libraries.
+
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+
+from ansys.mapdl.core import launch_mapdl
+
+# Start MAPDL.
+mapdl = launch_mapdl()
+
+
+###############################################################################
+# Initiate Pre-Processing
+# ~~~~~~~~~~~~~~~~~~~~~~~
+# Enter verification example mode and the pre-processing routine.
+
+mapdl.clear()
+mapdl.verify()
+mapdl.prep7()
+
```

Comment:
You could suppress the output with:
```
_ = mapdl.prep7()
```

Reason why the output is printing here is sphinx always prints the output at the end of each cell. For example:

```py
>>> a = 1
>>> a
1
```
However, this would produce no output:
```
>>> a = 1
```

Reply:
Maybe it is better then just store the last command

```python
mapdl.clear()
mapdl.verify()
_ = mapdl.prep7() #Underscore to suppress output.
```

Reply:
Yes, and do this on non-critical commands. We should want to see the output from some commands.